build_tools/crowdin.yaml

```diff
@@ -0,0 +1,23 @@
+project_identifier: kolibri
+api_key_env: CROWDIN_API_KEY
+
+base_path: ..
+
+preserve_hierachy: false
+
+files:
+  - source: '/kolibri/locale/en/LC_FRONTEND_MESSAGES/*.json'
```

Comment:
Just to verify that you've manually tested all this, because we've had some right laughs with crowdin config before!

Reply:
true. I did _not_ fully test my initial config work here https://github.com/learningequality/kolibri/pull/3176/commits/379a23b502441aa6f2c198e308abf47b52c6e3e3

Reply:
Yes, all is tested, it works: http://kolibri.readthedocs.io/es/spanish-rtd-test/

Reply:
By testing, I think Richard and I are referring to the process for uploading and downloading Kolibri strings. I don't suppose you tested that process?

Reply:
>  I don't suppose you tested that process?

Yes I did! :) But I will test it again, because some stuff is changing after `docs` got migrated

Reply:
'twas tested again :)